@ndlib/gatsby-theme-marble/src/assets/i18n/json/text.default.en.json

```diff
@@ -6,6 +6,29 @@
   "noUserPage": "The user <1>{{ userName }}</1> does not exist or you are not authorized to view this page.",
   "userLogoutPage": "You have been successfully logged out.",
   "userPage": {
-    "loginPrompt": "<0>Log in</0> to access and edit your own content."
+    "loginPrompt": "<0>Log in</0> to access and edit your own content.",
```

Comment:
I don't see this one implemented. Just making sure it wasn't missed.

Reply:
This part was already implemented in App/Pages/User/UserLayout/PromptLogin